lighthouse-core/report/html/report-styles.css

```diff
@@ -61,8 +61,9 @@
   --navitem-hpadding: var(--body-font-size);
   --navitem-vpadding: calc(var(--navitem-line-height) / 2);
   --lh-score-highlight-bg: hsla(0, 0%, 90%, 0.2);
-  --lh-score-icon-background-size: 24px;
   --lh-group-icon-background-size: var(--lh-score-icon-background-size);
+  --lh-info-icon-background-size: 20px;
+  --lh-score-icon-background-size: 24px;
```

Comment:
just -lh-score-icon-size ?

Reply:
btw ur commenting on something i just reordered

i added `--lh-info-icon-background-size`

could rename stuff after all this tho